docs/topics/js/js-ir-compiler.md

```diff
@@ -72,6 +71,37 @@ kotlin {
 }
 ```
 
+## Lazy initialization of top-level properties
+
+> Lazy initialization of top-level properties is [Experimental](components-stability.md). It may be dropped or changed at any time.
+> Opt-in is required (see the details below), and you should use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-44320).
+>
+{type="warning"}
+
+For better application startup performance, the Kotlin/JS IR compiler offers an option to initialize top-level properties
+lazily. This way, the application loads without initializing all the top-level properties used in its code. It initializes
+only the ones needed at startup; other properties get initialized later when the code that uses them is executed. 
+
+We're still working on this feature, so it's experimental and requires an opt-in. To use the lazy initialization of top-level
```

Comment:
IDK do we really need to highlight that we are still working on it?
Maybe we should tell it in the components-stability.md doc page (experimental section)?